db/migrate/20191126080621_change_data_type_for_admin.rb

```diff
@@ -0,0 +1,16 @@
+class ChangeDataTypeForAdmin < ActiveRecord::Migration[6.0]
+  def self.up
+    add_column :users, :role, :integer, default: 0
+
+    User.all.each do |user|
+      if user.admin
+        user.role = 1
```

Comment:
This could be written as `user.role = :admin or user.admin!`. And you are missing else that sets user to role `user` if it is not admin